packages/block-library/src/post-time-to-read/edit.js

```diff
@@ -93,6 +126,45 @@ function PostTimeToReadEdit( { attributes, setAttributes, context } ) {
 					} }
 				/>
 			</BlockControls>
+			<InspectorControls>
+				<ToolsPanel
+					label={ __( 'Settings' ) }
+					resetAll={ () => {
+						setAttributes( {
+							displayAsRange: false,
+						} );
+					} }
+					dropdownMenuProps={ dropdownMenuProps }
+				>
+					<ToolsPanelItem
+						isShownByDefault
+						label={ _x(
+							'Display as range',
+							'Turns reading time range display on or off'
+						) }
```

Comment:
```suggestion
						label={ __( 'Display as range' ) }
```

I think the meaning of the sentence can be conveyed without providing context options.